docs/docs/docs/deployment/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: docs
+title: Deployment
+url: /docs/deployment
+---
+
+# Deployment
+
+## Requirements
+
+In order to deploy compendium, it is necessary to provide an storage for both, metadata and protocols. **Each one will use its own database** to store the necessary information.
```

Comment:
Maybe we can mention here that, for simplicity reasons, both metadata and storage can be within the same postgres database, but have their own tables?

```suggestion
In order to deploy Compendium, it is necessary to provide an storage for both metadata and protocols. **Each one will use its own database** to store the necessary information.
```